7_react/react1/week1/vite-project/src/components/listingItems.jsx

```diff
@@ -0,0 +1,15 @@
+
+const ListingItems = (props) => {
+  return (
+    <div>
+        <ul>
```

Comment:
unordered list `<ul>` element should be wrapping `all` children, you should use it instead of the `div` element in the `IteratingTodoItemsArray`

Reply:
Also you do not need `div` tag here as well

Reply:
Both done :)

Reply:
I cannot see it, please first push the commit with the changes :) Thank you Katy

Reply:
@BrantalikP Sorry, I first commented and then pushed changes.  Will not repeat that. Thank you :)